lib/pester.rb

```diff
@@ -71,6 +71,7 @@ def self.retry_action(opts = {}, &block)
         end
       end
     end
+    nil
```

Comment:
yeah nil makes more sense then at least it's deterministic

On Sat, Apr 4, 2015 at 2:05 PM, Marc Bollinger <notifications@github.com>
wrote:

> In lib/pester.rb
> <https://github.com/lumoslabs/pester/pull/8#discussion_r27772133>:
>
> > @@ -71,6 +71,7 @@ def self.retry_action(opts = {}, &block)
> >          end
> >        end
> >      end
> > +    nil
>
> You mentioned it before, if the post-retry exhaustion action doesn't
> raise, it used to return the number of retries it was supposed to make,
> which was dumb. when i was writing #7
> <https://github.com/lumoslabs/pester/pull/7>, I realized that in the
> trivial case of having it try 0 or "fewer" times, it would just do nothing
> and return that number. nil seems like it makes more sense. this should
> only be returned if max_attempts is < 1 or when on_max_attempts_exceeded
> is called with something that doesn't reraise.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lumoslabs/pester/pull/8/files#r27772133>.
>